calendra/usa/core.py

```diff
@@ -75,6 +77,62 @@ def observance_shift(self):
     # Some regional variants
     include_mardi_gras = False
 
+    # Shift day mechanism
+    # These days won't be shifted to next MON or previous FRI
+    shift_exceptions = (
+        # Exemple:
+        # (11, 11),  # Veterans day won't be shifted
+    )
+
+    def shift(self, holidays, year):
```

Comment:
Probably this shift mechanism can be replaced by Holiday instances that are aware of their shift patterns. Was this new in the recent changes? I don't see it called anywhere, so maybe it can be eliminated.

Reply:
I must be going mad. I don't see this code in that file?